stock_valuation_fifo_lot/wizard/stock_valuation_layer_revaluation.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import Command, _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import float_compare, float_is_zero
+
+
+class StockValuationLayerRevaluation(models.TransientModel):
+    _inherit = "stock.valuation.layer.revaluation"
+
+    current_value_lot_svl = fields.Float(
```

Comment:
I think we can remove the `_svl` part as we are not looking at SVL records.
```suggestion
    current_value_lot = fields.Float(
```